.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F41E Report an error or problem"
+about: Before you start, check whether a similar issue was already reported. Search via https://github.com/redaxo/redaxo/issues
+
+---
+
+**Description / Beschreibung**
+
+**Affected versions / Verwendete Versionen**
```

Comment:
Hier könnte man zukünftig eventuell auf den Systemreport hinweisen, mit dem man die Infos am einfachsten angeben kann.

Reply:
Haben wir notiert - war auch in meinem Ausgangszext drin